code/libraries/koowa/components/com_koowa/database/table/templates.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Nooku Framework - http://nooku.org/framework
+ *
+ * @copyright   Copyright (C) 2007 - 2014 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        https://github.com/nooku/nooku-framework for the canonical source repository
+ */
+
+/**
+ * Database table for Joomla templates
+ *
+ * @author  Dave Li <https://github.com/daveli>
+ * @package Koowa\Component\Koowa\Database\Table
+ */
+class ComKoowaDatabaseTableTemplates extends KDatabaseTableAbstract
```

Comment:
@daveli I would consider renaming this to ComKoowaModelTableThemes instead of Templates. This will be less confusing as we already use the name templates in the framework which means something different. 

@ercanozkaya In the platform this database table has been renamed to 'templates' not 'template_styles' we will need to add a check here to set the name correctly.